### ISSUE TITLE
Webchess in spanish

### ISSUE THREAD

**Issue by jlmurillo** [benjamw/chess#5]
I have translated almost 85% of WebChess to spanish, but not to change the notation (Nxf4-> Cxf4, Qb8-> Db8, ...) and "white" and "black" in some pages by shredding the code or database.
Where you advise me that I can change that.
Thank you

**Comment by benjamw** [comment]
I don't have the time or desire to go through the project looking for safe places to translate, but these places are safe to translate:
* anything that is inside an HTML tag, example: <tag>white</tag>
* anything in `/help` directory

Just try things out, and if they break, then change them back.

**Comment by jlmurillo** [comment]
Yeah, but I'm not a programmer and I can hardly tell when it is the name 
of a variable, the content, tag, ... but keep trying with this track. 
Thank you.
